torchci/rockset/metrics/__sql/master_commit_red.sql

```diff
@@ -27,7 +27,7 @@ with any_red as (
                 job.name != 'ciflow_should_run'
                 AND job.name != 'generate-test-matrix'
                 AND workflow.event != 'workflow_run' -- Filter out worflow_run-triggered jobs, which have nothing to do with the SHA
-                AND push.ref = 'refs/heads/master'
+                AND push.ref IN ('refs/heads/master', 'refs/heads/main')
```

Comment:
This one is interesting--I think you may need to rebase as the query has changed since your change. 

e5003789ace2ca6a is the version.

Reply:
What's really interesting is that it says my branch is up to date with main

Reply:
Yes, main is the default branch for this. Huh. Does using the hash I gave work?

Reply:
It seems to work 
![image](https://user-images.githubusercontent.com/12445270/163866923-ccbb4961-160b-460d-ad0d-810333bdcf47.png)